instructure/tinymce-a11y-checker
# Make links for more information underlined

This also updates using InstUI to 5.37

closes CORE-2097

Test Plan:
  - yarn start
  - Run the a11y checker
  - Click the (?)
  - The "Learn more about..." link should be underlined
